.github/workflows/wbstack.sync.yml

```diff
@@ -16,5 +16,9 @@ jobs:
         run: ./sync.sh
       - name: Show `git diff`` output
         run: git diff
-      - name: Check only composer autoloaders changed
-        run: diff -Z ./.github/workflows/wbstack.sync.expect.txt <(git status --porcelain=v1)
+      - name: Check only composer changed
+        run: |
+          NUM_CHANGES=$(git status --porcelain=v1 -- ':!dist/vendor/composer' ':!dist/vendor/autoload.php')
```

Comment:
This might be better named `CHANGES` since the variable contains all the string output of the status command